src/OpenTelemetry.Exporter.Geneva/GenevaBaseExporter.cs

```diff
@@ -89,4 +90,19 @@ internal static int AddPartAField(byte[] buffer, int cursor, string name, object
         cursor = MessagePackSerializer.Serialize(buffer, cursor, value);
         return cursor;
     }
+
+    internal static int AddPartAField(byte[] buffer, int cursor, string name, Span<byte> span)
```

Comment:
Minor: consider using `value` to reflect the intention (the `value` of the field).

```suggestion
    internal static int AddPartAField(byte[] buffer, int cursor, string name, Span<byte> value)
```